infrastructure/modules/armonik/locals.tf

```diff
@@ -152,6 +162,19 @@ locals {
     } : key => value if !contains(values(value), "")
   }
 
+  pod_authentication_in_database_credentials = {
```

Comment:
You can rename the local variable "pod_partitions_in_database_credentials" by "database_credentials" and use it in : 
* "partitions_in_database" job
* "authentication_in_database" job
* "partitions_in_database" cron job.